cypress/e2e/certificate-provider.cy.js

```diff
@@ -0,0 +1,124 @@
+describe('Certificate provider task', () => {
+    beforeEach(() => {
+        cy.visit('/testing-start?redirect=/task-list');
+    });
+
+    it('can be done later', () => {
+        cy.contains('li', "Choose your certificate provider")
+            .should('contain', 'Not started')
+            .find('a')
+            .click();
+
+        cy.injectAxe();
```

Comment:
Assuming you already tried this but I thought we only needed to inject axe after the initial visit and it would be available for the remainder of the test. Is it required after navigating to each new page?

Reply:
yep, I assume it puts some JS on the page so it needs injecting whenever a load happens